pkg/cli/admin/upgrade/status/examples/4.14.1-degraded.detailed-output

```diff
@@ -12,12 +12,8 @@ ip-10-0-53-40.us-east-2.compute.internal    Outdated     Pending   4.14.0-rc.3
 ip-10-0-92-180.us-east-2.compute.internal   Outdated     Pending   4.14.0-rc.3   ?     
 
 = Worker Upgrade =
-
-= Worker Pool =
-Worker Pool:     worker
-Assessment:      Pending
-Completion:      0%
-Worker Status:   3 Total, 3 Available, 0 Progressing, 3 Outdated, 0 Draining, 0 Excluded, 0 Degraded
+Worker Pool   Assessment   Completion   Status
```

Comment:
I was following the card but ... do we want to use all-capital letters for the table header? (to be aligned with the "Worker Pool Nodes" table below?

Reply:
yeah, let's all-cap for consistency